modules/force_rtl.py

```diff
@@ -29,9 +28,6 @@ def force_rtl(drone: dronekit.Vehicle) -> bool:
     # Generate and set RTL command
     rtl_command = generate_command.return_to_launch()
 
-    # Change drone mode
-    drone.mode = dronekit.VehicleMode("RTL")
-
     # Utilize upload_command function to give RTL command to drone
     result = upload_commands.upload_commands(drone, [rtl_command], DRONE_TIMEOUT)
```

Comment:
instead of using upload_commands module, you should use the flightcontroller's upload_command function!